src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventSchema.scala

```diff
@@ -79,6 +79,37 @@ object CassandraPopularTopics {
   }
 }
 
+object CassandraConjunctiveTopics {
+  def apply(item: Event): Seq[ConjunctiveTopicAggregate] = {
+    val keywords = item.computedfeatures.keywords
+    val keywordPairs = keywords.combinations(2).flatMap(combination=>Seq(
```

Comment:
That's what I worried about. If there's only one term hit then there should be one record in this table with the total mention count for that term.